docs/FAQ.md

```diff
@@ -4,30 +4,22 @@
 
-## Why does `Relude` use peer dependencies for everything, rather than hard production dependencies?
-
-The way that `npm` installs dependencies was designed specifically for the JavaScript ecosystem,
-where it made sense for each library to install its own set of dependencies.  That way, each dependency could be installed with some level of confidence that it would work with its own set of dependencies, and not be affected by other libraries installing other likely incompatible versions of the same dependencies.
-
-However, in a statically-typed language, this doesn't work so well, because the compiler seeks to align the shared types across modules and libraries.  E.g. if two libraries `A` and `B` depend on some module `C`, the compiler will expect to find a single version of the library `C`, so that it can compile `A` and `B` with the same types exposed by `C`.  If this were not the case, it would be difficult for libraries to work together with the same shared types.
-
-We use peer dependencies in all `Relude` libraries, so that the top-level host library or application can ultimately control what single version of each dependency is installed.  E.g. `Relude` depends on `bs-bastet` as a peer dependency, so any library using `Relude` must also explicitly install `bs-bastet`, as it won't be installed automatically with `Relude`.
 
 ## What's the difference between `Relude.Result` and `Belt.Result`, `Relude.Option` and OCaml stdlib `option`, `Relude.Js.Json` and `Js.Json`, etc.?
 
 `Relude` has many modules that augment existing types from `Belt` or the OCaml standard library, which are both available by default within Melange/ReasonML projects.  Sometimes we alias the types from `Belt` or the OCaml stdlib, and sometimes our corresponding modules simply add additional functions for these common types.
 
 `Relude` also has many of its own types, which have their own modules, like `Relude.AsyncResult`, `Relude.ReaderT`, etc.
 
-## I'm getting strange errors about types like `array(string)` not being the same as `BsBastet.Array.Foldable.t`
+## I'm getting strange errors about types like `array(string)` not being the same as `Bastet.Array.Foldable.t`
 
 If you are trying to compile and get an error like:
 
 ```reasonml
-This has type array(t) but somewhere wanted BsBastet.Array.Foldable.t(string)`
+This has type array(t) but somewhere wanted Bastet.Array.Foldable.t(string)`
 ```
 
-you likely need to add `bs-bastet` to your `bs-dependencies` in your `bsconfig.json` file.  This error occurs because Melange is not able to find the types defined in the `BsBastet` module, so it can't determine that `array(t)` is the same type as `BsBastet.Array.Foldable.t(string)`.
+you likely need to add `mel-bastet` to your `bs-dependencies` in your `bsconfig.json` file.  This error occurs because Melange is not able to find the types defined in the `Bastet_js` module, so it can't determine that `array(t)` is the same type as `Bastet.Array.Foldable.t(string)`.
```

Comment:
Similar to my comment below, this is another whole section that can probably just be removed, as weird errors related to bucklescript dependency config shouldn't really apply anymore.